### PR TITLE
Hide footer on mobile devices

### PR DESCRIPTION
**Pull request by wozniaczek123**

### CODE REVIEW

File: patches/hideFooter.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@media screen and (max-width: 1023px) {
+    footer {
+        display: none;
```
Comment (PanLiszka) [inline patches/hideFooter.css:3]: This style is overwriten by page css

```suggestion
        display: none !important;
```

**Review by PanLiszka — CHANGES_REQUESTED** [review]

**Review by banocean — CHANGES_REQUESTED** [review]
I think we might not want to do it on `https://eduvulcan.pl/*` their footer doesn't make experience worse

**Comment by banocean** [comment]
> I think we might not want to do it on `https://eduvulcan.pl/*` their footer doesn't make experience worse

But the bug @PanLiszka mentioned might be a way to implement that :thinking:. We prob still want a comment explaining this doesn't work on purpose.
ref: https://github.com/banocean/ifv/pull/27#discussion_r1739799207

**Comment by PanLiszka** [comment]
Bug that it's not removing footer from `eduvulcan.pl` is I guess a feature? Main page look good with footer and it works on register page :)

File: patches/hideFooter.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@media screen and (max-width: 1023px) {
+    footer {
+        display: none !important;
```
Comment (banocean) [inline patches/hideFooter.css:3]:
```suggestion
        display: none;
```
We're sticking to what we said in the comment

**Review by banocean — APPROVED** [review]

**Pull request merged**